front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/EntityTagsController.groovy

```diff
@@ -87,7 +87,7 @@ class EntityTagsController {
     }
 
     taggedEntityDAO.bulkImport(tags)
-    return findAllByIds(tags.findResults { it.id })
```

Comment:
This is actually a fairly significant contributor to overall slowness.

The entity tag objects themselves aren't changing aside from `lastModified` and `lastModifiedBy` which aren't a deal breaker. As it stands the object references are being updated directly so they're going to be picked up here anyhow.

Reply:
yeah, this makes sense!